packages/common/src/dto/personal-information.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsString, Length, Matches } from 'class-validator';
+
+export class PersonalInformationDTO {
+  @IsString()
+  @Length(1, 255)
+  firstName!: string;
+
+  @IsString()
+  @Length(1, 255)
+  lastName!: string;
+
+  @IsString()
+  @Matches(/^[ABCEGHJKLMNPRSTVXY]\d[ABCEGHJ-NPRSTV-Z][\s-]?\d[ABCEGHJ-NPRSTV-Z]\d$/i)
```

Comment:
I believe the generated error message is not very user-friendly. This could do a custom message.

Reply:
100%, the full validator implementation will come in the next PR, I should have left this change out of this PR